lib/aur-build

```diff
@@ -47,7 +48,7 @@ source /usr/share/makepkg/util/util.sh
 source /usr/share/makepkg/util/message.sh
 source /usr/share/makepkg/util/parseopts.sh
 
-if [[ -t 2 && ! -o xtrace ]]; then
+if [[ ( -t 1 && -t 2 && ! -o xtrace && $AUR_COLOR == auto ) || $AUR_COLOR == always ]] ; then
```

Comment:
The manual update mentions `AUR_COLOR=never` or any unrecognized value. Should `never` be explicitely checked here, or the mention removed from `aur(1)`?

Reply:
The documentation is the API, the fact that we don't need to explicitly check for the 'never' value is just an implementation detail.

But I guess we can check for the 'never' value just for the sake of clarity.

``` bash
if [[ ( $AUR_COLOR == auto && -t 1 && -t 2 && ! -o xtrace ) || $AUR_COLOR == always || $AUR_COLOR != never ]] ; then
    colorize
fi
```
What do you think?

Reply:
`AUR_COLOR=${AUR_COLOR:-auto}` should be `AUR_COLOR=${AUR_COLOR-auto}`.

Reply:
OK, let's keep the original approach then. (Though I'm not sure how `AUR_COLOR == always` is redundant, if you say "the last two tests".)

Reply:
Poor choice of words on my part. Only the last test is redundant. What I was trying to say is that `$AUR_COLOR == always && $AUR_COLOR != never`will only evaluate to true if the value is `always`, which is already being tested by the first part.

Reply:
Yeah, if we accept any values for `AUR_COLOR` with only auto/always enabling color, such a check can only be redundant. (I guess the alternative is an interface where other values than auto/always/never cause an error, but eh.)